### PR TITLE
feat/pd-1382

### PR DESCRIPTION
**Pull request by CarlaCostea**
https://illuminate.atlassian.net/browse/PD-1382

### CODE REVIEW

File: packages/inline-dropdown/src/index.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ export default class RootInlineDropdown extends HTMLElement {
         rationale: this._model.rationale,
         teacherInstructions: this._model.teacherInstructions,
         disabled: this._model.disabled,
+        displayType: this._model.displayType,
```
Comment (andreeapescar) [inline packages/inline-dropdown/src/index.js:46]: Is this related to another ticket?

Comment (CarlaCostea) [reply packages/inline-dropdown/src/index.js:46]: yes

**Comment by andreeapescar** [comment]
Please fix conflicts, @CarlaCostea
